plugins/dispatch-webmentions/mention.js

```diff
@@ -45,7 +47,13 @@ export class Mention {
       throw new Error(`Unexpected status: ${res.status}`);
     }
 
-    const { window } = new JSDOM(await res.text(), { url: res.url, contentType: res.headers.get('content-type') });
+    const contentType = res.headers.get('content-type') || 'text/html';
+
+    if (contentType !== 'text/html' && contentType !== 'application/xhtml+xml') {
```

Comment:
It might be interesting to consider other content types (gemini for example), but different parsers would be needed for those. For now I've reduced the allowable types to HTML and XHTML.